### ISSUE TITLE
2022.5-beta1 - interface (GUI) doesn't show up

### ISSUE THREAD

**Issue by resiehnnes** [mullvad/mullvadvpn-app#3981]
Operating system: Fedora 36, Gnome 42.5, x11
App version: 2022.5-beta1

After updating from stable 2022.4 to 2022.5-beta1 Mullvad's interface (GUI) doesn't show up after opening it from tray icon. Tried using x11 and Wayland protocols, same problem.

**Comment by raksooo** [comment]
Thanks for reporting! I'm unfortunately not able to reproduce this. What I've tried:
* Running Fedora 36
* Using Gnome Desktop 42.4 with gnome-shell 42.5
* Using [AppIndicator and KStatusNotifierItem Support](https://extensions.gnome.org/extension/615/appindicator-support/) for tray icon support
* Opening the Mullvad  app
* Pressing `x` to hide the window
* Pressing the tray icon and then "Open Mullvad VPN"'

Did it work for you with Gnome Desktop 42.4?
Which Gnome shell extension are you using for tray support?

**Comment by resiehnnes** [comment]
Hi Oskar. I have completely identical configuration as yours. So what I tried so far:

- Updating Mullvad from 2022.4 to 2022.5-beta1 in terminal
- Removing and installing 2022.5-beta1 from scratch

Same results, interface doesn't show up nor from system tray, nor from launcher. 2022.4 works with no problems.

**Comment by raksooo** [comment]
What is the output if you run it from the terminal? `/opt/Mullvad\ VPN/mullvad-vpn`
Does it make a difference if you run it with these flags? `/opt/Mullvad\ VPN/mullvad-vpn --disable-gpu --no-sandbox`

**Comment by resiehnnes** [comment]
So after running Mullvad from terminal I noticed a warning about failing to load image (icon) from 'Papirus icon pack' which I am using. I tried to set default icons (Adwaita) in Gnome Tweaks and Mullvad's gui appeared after restarting the app.

> What is the output if you run it from the terminal? `/opt/Mullvad\ VPN/mullvad-vpn`

[2022-09-29 17:45:07.829][verbose] Chromium sandbox is enabled
[2022-09-29 17:45:07.831][info] Running version 2022.5-beta1
[2022-09-29 17:45:07.914][info] Detected locale: en-US
[57245:0929/174507.929075:ERROR:browser_main_loop.cc(271)] GLib-GObject: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[2022-09-29 17:45:07.945][info] Connected to the daemon
[2022-09-29 17:45:08.044][info] Autoconnect the tunnel
(node:57245) UnhandledPromiseRejectionWarning: Error: Failed to load image from path '/usr/share/icons/Papirus/128x128/apps/mullvad-vpn.svg'
    at UserInterface.setWindowIcon (/opt/Mullvad VPN/resources/app.asar/build/src/main/user-interface.js:58:70)
    at ApplicationMain.onReady (/opt/Mullvad VPN/resources/app.asar/build/src/main/index.js:168:40)
(Use `mullvad-gui --trace-warnings ...` to show where the warning was created)
(node:57245) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

> Does it make a difference if you run it with these flags? `/opt/Mullvad\ VPN/mullvad-vpn --disable-gpu --no-sandbox`

No difference if I keep using 3rd party's icon pack.
